src/api/two/models.rs

```diff
@@ -19,7 +19,7 @@ pub struct Meta {
     pub next: Option<String>,
     pub previous: Option<String>,
     #[serde(rename = "self")]
-    pub _self_: String,
+    pub self_url: String,
```

Comment:
@supaiku0 I changed this, but I'm not sure about the best approach, what do you think?

Reply:
Your change is fine, just picked a random name because it is reserved. :thinking: